Library/Homebrew/os/mac.rb

```diff
@@ -50,7 +50,9 @@ def self.latest_sdk_version
 
     sig { returns(String) }
     def self.preferred_perl_version
-      if version >= :big_sur
+      if version >= :sonoma
+        "5.34"
```

Comment:
`perl -V` gives me `This is perl 5, version 30, subversion 3 (v5.30.3) built for darwin-thread-multi-2level`.

Reply:
Yeah. We don't actually use `/usr/bin/perl` anymore - we now use `/usr/bin/perl5.XX`. I think both 5.30 and 5.34 exist.

I lean on the newer one because we got caught out with Big Sur where 5.18 and 5.30 both shipped (and 5.18 was the default) but 5.18 was removed in 11.3.

5.34 technically existed in Ventura too but `/System/Library/Perl/Extras/5.34` did not, so it wasn't a full installation.

Reply:
Ah right, makes sense. Yes, that's there and the paths in `@INC` exist too.